deploy/src/main/sandbox-k8s/common.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#*******************************************************************************
+# Copyright (c) 2021 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+
+set -ue
+
+KUBECONFIG=${2:-"--kubeconfig /etc/rancher/k3s/k3s.yaml"}
```

Comment:
will this *always* be provided as the second parameter? Is there a mandatory first parameter?

Reply:
I failed to change that when removing the first one. I don't think we need to have this configurable by parameters.

Reply:
I changed it to a constant value.